src/lib/fees/model.ts

```diff
@@ -61,20 +52,44 @@ export const calculateFees: CalculateFeeBps = ({ tradeAmountUsd, foxHeld }) => {
         ),
       ),
   )
-
-  const feeBps = BigNumber.maximum(
-    feeBpsBeforeDiscount.multipliedBy(bn(1).minus(foxDiscountPercent.div(100))),
+  const feeBpsRaw = BigNumber.maximum(
+    feeBpsBeforeDiscountRaw.multipliedBy(bn(1).minus(foxDiscountPercentRaw.div(100))),
     bn(0),
   )
+
+  const feeBpsBeforeDiscount = feeBpsBeforeDiscountRaw.decimalPlaces(0)
+  const feeBpsAfterDiscount = feeBpsRaw.decimalPlaces(0)
```

Comment:
Do we want to `dp(0)` this later on, so we have a more accurate discount percentage?